openstack/networking/v2/subnets/requests.go

```diff
@@ -108,6 +108,7 @@ type CreateOpts struct {
 	TenantID        string
 	AllocationPools []AllocationPool
 	GatewayIP       string
+	NoGateway       bool
```

Comment:
If the default is reasonable, `bool` is fine, and we use it elsewhere in the library like that. The ~~Neturon~~ OpenStack docs aren't good, so I'm sure we just didn't know that enabling DHCP was the default. Otherwise we probably would've named it `DisableDHCP` and made it a `bool`.